modules/Bio/EnsEMBL/Analysis/Hive/Config/TranscriptSelection.pm

```diff
@@ -0,0 +1,954 @@
+
+    genome_dumps => catdir( $self->o('output_path'), 'genome_dumps' ),
+    # This one is used in replacement of the dna table in the core db, so where analyses override slice->seq. Has simple headers with just the seq_region name. Also used by bwa in the RNA-seq analyses. Not masked
+    faidx_genome_file => catfile( $self->o('genome_dumps'), $self->o('species_name') . '_toplevel.fa' ),
+
+    'layering_input_gene_dbs' => [
+      $self->o('genblast_nr_db'),
+      $self->o('genblast_rnaseq_support_nr_db'),
+      $self->o('rnaseq_for_layer_nr_db'),
+      $self->o('selected_projection_db'),
+      $self->o('ig_tr_db'),
+      $self->o('best_targeted_db'),
+      $self->o('long_read_final_db'),
+    ],
+
+    utr_donor_dbs => [
+      $self->o('cdna_db'),
+      $self->o('rnaseq_for_layer_db'),
+      $self->o('long_read_final_db'),
+    ],
+
+    utr_acceptor_dbs => [
+      $self->o('layering_db'),
+    ],
+
+    'utr_biotype_priorities' => {
+      'rnaseq' => 2,
+      'cdna'   => 1,
+    },
+
+    'cleaning_blessed_biotypes' => {
+      'pseudogene'           => 1,
+      'processed_pseudogene' => 1,
+      'IG_C_gene'            => 1,
+      'IG_V_gene'            => 1,
+      'TR_C_gene'            => 1,
+      'TR_D_gene'            => 1,
+      'TR_V_gene'            => 1,
+      'lncRNA'               => 1,
+    },
+
+    ensembl_analysis_script => catdir( $self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts' ),
+    remove_duplicates_script_path => catfile( $self->o('ensembl_analysis_script'), 'find_and_remove_duplicates.pl' ),
+    remove_small_orf_script => catfile( $self->o('ensembl_analysis_script'), 'genebuild', 'remove_small_orf.pl' ),
+
+    # cutoffs for removing small_orf genes
+    'small_orf_cutoff' => '100',
+    'intron_cutoff'    => '75',
+
+########################
+# Extra db settings
+########################
+
+    'num_tokens' => 10,
+
+########################
+# Executable paths
+########################
+
+    'blast_type' => 'ncbi',    # It can be 'ncbi', 'wu', or 'legacy_ncbi'
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# No option below this mark should be modified
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+#
+
+########################
+# db info
+########################
+
+    'cdna_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_cdna_' . $self->o('release_number'),
+      -host   => $self->o('cdna_db_server'),
+      -port   => $self->o('cdna_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genblast_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_genblast_' . $self->o('release_number'),
+      -host   => $self->o('genblast_db_server'),
+      -port   => $self->o('genblast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
```

Comment:
This one can be removed. The pipeline is using genblast_nr_db which has the same information but not the duplicates

Reply:
It's used in 'create_toplevel_slices'. Do I need to replace it with genblast_nr_db?